### DISCUSSION TITLE
When will Rsbuild V1.0 be released?

### DISCUSSION THREAD

**Discussion by halomai** [web-infra-dev/rsbuild#3249] [General]
想问一下这边大概什么时候发布Rsbuild的1.0正式版本呢，我们这边有项目想等1.0正式版发布之后进行迁移

**Comment by chenjiahan** [comment]
Rspack v1.0 will be released on 8.28, and then we will move forward with Rsbuild 1.0, which will be released in September 2024. In addition, Rsbuild 1.0 beta is now stable enough to be used in production.

> Rspack v1.0 将在 8.28 号发布，然后我们会推进 Rsbuild 1.0 的工作，Rsbuild 1.0 预计在 2024 年 9 月发布。另外目前 Rsbuild 1.0 beta 已经足够稳定，可以用于生产。
